src/gsi/displace_wrf.F90

```diff
@@ -0,0 +1,495 @@
+#ifdef FCA_REF_MOD
```

Comment:
Same as other comments. Please remove unused lines, add a proper documentation in the head. I will review again clean up.

Reply:
ok, comments cleaned up